### DISCUSSION TITLE
Authorization management - required policy for request

### DISCUSSION THREAD

**Discussion by Noste92** [mia-platform/community#425] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

PaaS

### Description

Hi,

I'd like to configure a policy of response filtering type in the related section:
![image](https://github.com/mia-platform/community/assets/14860750/f36cd5bd-b298-45f7-a3d8-a9f9ee459254)
In this case I don't need a `policy for request` because I need only to modify the response for specific clients, so I don't understand why the `policy for request` is a required field. 
Am I missing something?

Thanks

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, it's a design choice from [rönd](https://github.com/rond-authz/rond) which requires the ingress policy to always be defined.

You can always create a policy that is always true and re-use it in your use case


```rego
package policies

allow_all {
    true
}
```

**Reply by Noste92** [reply]
Ok, thank you @fredmaggiowski :+1:
